### DISCUSSION TITLE
Update Package Microsoft.AspNetCore.Authentication.OAuth

### DISCUSSION THREAD

**Discussion by Joshlo** [dotnet/aspnetcore#23917] [General] [answered]
Hi,

Is it possible to update https://www.nuget.org/packages/Microsoft.AspNetCore.Authentication.OAuth/ to newest version? (2.2.0 -> 3.1.5)

As it is right now, if I want to use the newest version of OAuthHandler I have to download Google, Facebook, Twitter or OpenConnect version, just to get access to it.

Best regards
Kenneth

**Comment by Tratcher** [comment, marked as answer]
https://docs.microsoft.com/en-us/aspnet/core/migration/22-to-30?view=aspnetcore-3.1&tabs=visual-studio#remove-obsolete-package-references
https://docs.microsoft.com/en-us/aspnet/core/migration/22-to-30?view=aspnetcore-3.1&tabs=visual-studio#framework-reference

The OAuth dll is now bundled as part of the framework, you no longer need a package dependency for it.
